CHANGELOG.md

```diff
@@ -2,6 +2,9 @@
 
 This is a list of high-level changes for each release of AWX. A full list of commits can be found at `https://github.com/ansible/awx/releases/tag/<version>`.
 
+## 16.0.0 (TBD)
+- Removed support for Mercurial-based project updates - https://github.com/ansible/tower/issues/4727
```

Comment:
Might want to hold off and rebase my PR: https://github.com/ansible/awx/pull/8723

Reply:
Actually, you go first and I'll deal w/ the conflict.